common/djangoapps/student/tests/test_admin_views.py

```diff
@@ -347,6 +347,12 @@ def test_unicode_username(self):
         )
         self.assertEqual(str(LoginFailures.objects.get(user=self.user2)), 'Zażółć gęślą jaźń: 2 - -')
 
+    @override_settings(FEATURES={'ENABLE_MAX_FAILED_LOGIN_ATTEMPTS': True})
+    def test_feature_enabled(self):
```

Comment:
**Note to reviewers:** I added a test method (that was missing) to check for a successful response when the feature is enabled.